sapling/src/webviews/components/Navbar.tsx

```diff
@@ -7,27 +7,24 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 
 const Navbar = ({ rootFile }: any) => {
-  // onChange function that will send a message to the extension when the user selects a file
-  const fileMessage = (e: any) => {
-    const filePath = e.target.files[0].path;
-    // Reset event target value to null so the same file selection causes onChange event to trigger
-    e.target.value = null;
-    if (filePath) {
-      tsvscode.postMessage({
+  
+  const fileMessage = () => {
+    // tell the extension that the user wants to open a file
+    tsvscode.postMessage({
         type: "onFile",
-        value: filePath
-      });
-    }
+        value: null
+    });
   };
 
   // Render section
   return (
     <div className="navbar">
-      <input type="file" name="file" id="file" className="inputfile" onChange={(e) => {fileMessage(e);}}/>
+      <button id="file" className="infputFile" onClick={() => fileMessage()}>
```

Comment:
Typo. Can you update infputFile to inputFile?

Reply:
done